website-1.x/docs/getting-started-docker.md

```diff
@@ -21,6 +21,8 @@ To run the local web server:
 
    This will start a docker container with the image `docusaurus-doc`. To see more detailed container info run `docker ps` .
 
+To access docusaurus from outside the docker container you must add the `--host` flag to the `docusaurus-start` command as described in: [API Commands](api-commands.md#docusaurus-start)
```

Comment:
```suggestion
To access Docusaurus from outside the docker container you must add the `--host` flag to the `docusaurus-start` command as described in: [API Commands](api-commands.md#docusaurus-start)
```

Reply:
I'd recommend adding an example with `--host 0.0.0.0` because the user might not understand how to configure this --host option?

Reply:
I didn't add the example because there already is an example if the reader follows the link. Though it is redundant and can get outdated (not probable).

Do you still think I should add it?